schema/search-xml.yaml

```diff
@@ -0,0 +1,163 @@
+          <opensearch:itemsPerPage xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/">1</opensearch:itemsPerPage>
+          <entry xmlns="http://www.w3.org/2005/Atom" xmlns:arxiv="http://arxiv.org/schemas/atom">
+            <id xmlns="http://www.w3.org/2005/Atom">http://arxiv.org/abs/hep-ex/0307015</id>
+            <published xmlns="http://www.w3.org/2005/Atom">2003-07-07T13:46:39-04:00</published>
+            <updated xmlns="http://www.w3.org/2005/Atom">2003-07-07T13:46:39-04:00</updated>
+            <title xmlns="http://www.w3.org/2005/Atom">Multi-Electron Production at High Transverse
+        Momenta in ep Collisions at HERA</title>
+            <summary xmlns="http://www.w3.org/2005/Atom">
+          Multi-electron production is studied at high electron transverse momentum in
+        positron- and electron-proton collisions using the H1 detector at HERA. The
+        data correspond to an integrated luminosity of 115 pb-1. Di-electron and
+        tri-electron event yields are measured. Cross sections are derived in a
+        restricted phase space region dominated by photon-photon collisions. In general
+        good agreement is found with the Standard Model predictions. However, for
+        electron pair invariant masses above 100 GeV, three di-electron events and
+        three tri-electron events are observed, compared to Standard Model expectations
+        of 0.30 \pm 0.04 and 0.23 \pm 0.04, respectively.
+        </summary>
+            <author xmlns="http://www.w3.org/2005/Atom">
+              <name xmlns="http://www.w3.org/2005/Atom">H1 Collaboration</name>
+            </author>
+            <arxiv:comment xmlns:arxiv="http://arxiv.org/schemas/atom">23 pages, 8 figures and 4 tables</arxiv:comment>
+            <arxiv:journal_ref xmlns:arxiv="http://arxiv.org/schemas/atom">Eur.Phys.J. C31 (2003) 17-29</arxiv:journal_ref>
+            <link xmlns="http://www.w3.org/2005/Atom"
+            href="http://arxiv.org/abs/hep-ex/0307015v1" rel="alternate" type="text/html"/>
+            <link xmlns="http://www.w3.org/2005/Atom" title="pdf"
+            href="http://arxiv.org/pdf/hep-ex/0307015v1" rel="related" type="application/pdf"/>
+            <arxiv:primary_category xmlns:arxiv="http://arxiv.org/schemas/atom"
+            term="hep-ex" scheme="http://arxiv.org/schemas/atom"/>
+            <category term="hep-ex" scheme="http://arxiv.org/schemas/atom"/>
+          </entry>
+        </feed>
+        ```
+
+      parameters:
+        - name: search_query
+          in: query
+          description: |
+            A string that represents a search query. Fields are specified by "field:value" and 
+            delimited with "AND", "OR", and "ANDNOT" boolean operators. Valid field names are:
+            ti (title), au (author), abs(abstract), co (comment), jr (journal reference), cat
+            (subject category), rn (report number), id (id - use id_list instead), all (all 
```

Comment:
I'm confused by `id (id - use id_list instead)`, seems to contain conflicting instructions for the valid field name.
Perhaps `id_list (arXiv ids, comma delimited)` or `id (arXiv id for single item only; to search within multiple articles use separate id_list parameter)`?
Is that a correct reading of what's going on?

Reply:
@JaimieMurdock can you field this?

Reply:
Going to mark this resolved, since it's tangential to the PR (despite showing up in the diff; not sure why that is). @JaimieMurdock let's make sure to address this in the course of your work on the API.